route/build/src/route_runoff.f90

```diff
@@ -121,6 +125,15 @@ program route_runoff
 type(KREACH),allocatable   :: wavestate(:,:)      ! wave states for all upstream segments at one time step
 integer(i4b),allocatable   :: wavesize(:,:)       ! number of wave for segments 
 integer(i4b)               :: LAKEFLAG            ! >0 if processing lakes
+! restart state variables
```

Comment:
could add comment to define each of these variables (eg rf_array()) and their dims